front/app/services/textingCampaigns.ts

```diff
@@ -81,3 +81,18 @@ export const deleteTextingCampaign = async (campaignId: string) => {
 
   return result;
 };
+
+export const sendTextingCampaign = async (campaignId: string) => {
+  const result = await streams.add(
+    `${apiEndpoint}/${campaignId}/send`,
+    // no body required, just POST to /send
+    {}
+  );
+
+  // campaign should be status "Sending" now, re-fetch the entity from BE
+  await streams.fetchAllWith({
+    apiEndpoint: [apiEndpoint, `${apiEndpoint}/${campaignId}`],
+  });
```

Comment:
Have you tried if it works without this?

Reply:
Yeah, in my testing, without the `apiEndpoint` the text list becomes stale (the campaign we just sent to SENDING is still in DRAFT) and without `apiendpoint/campaignId` the immediate redirect to the individual campaign page is stale (it shows "DRAFT" but should be "SENDING" since at this point we've had a successful response from the BE). 

I don't know enough about the streams to troubleshoot the actual cause, but I think since sending a campaign will be a relatively uncommon action, an extra two calls to make sure all the data is fresh is fine in this case